.formatter.exs

```diff
@@ -1,4 +1,6 @@
 # Used by "mix format"
 [
+  line_length: 120,
```

Comment:
We'd prefer the default. 120 is too long to me. If Elixir community decides that something else is better then I'd wait for them to make the change globally.